### PR TITLE
Feature/uche/transpile backend

### PR DESCRIPTION
**Pull request by Way2nnadi**
- backend code now can use es6+ syntax
- [Fix] image static loading
- code clean up to use es6 import/export and spread operator

### CODE REVIEW

File: routes/product/helpers.js [hunk 1]
```diff
@@ -12,28 +17,12 @@ module.exports = {
...
-            deletedAt,
-            name,
-            producerId,
-            quantity,
-            typeOfProduct,
-            updatedAt,
-        });
-
         try {
+            const productData = _pick(req.body, CREATE_KEYS);
```
Comment (EngrEric) [inline routes/product/helpers.js]: Wow, this is more than clean.
please, what is happening here?

File: routes/signUp/helpers.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import CONSTANTS from './constants';
+import models from '../../db/models/';
+
+const { Producer, User } = models;
+const { NO_EMAIL_PASSWORD, SIGN_UP_KEYS, USER_EXIST, SIGNED_UP  } = CONSTANTS;
+
+export default {
+    signUpUser: async (req, res) => {
+        // https://lodash.com/docs/4.17.11#pick
+        const userData = _pick(req.body, SIGN_UP_KEYS);
```
Comment (EngrEric) [inline routes/signUp/helpers.js]: i now understand.
wow, nice one sir

File: routes/product/helpers.js (hunk 1 above)
Comment (Way2nnadi) [reply routes/product/helpers.js]: clean but still makes intuitive sense

File: routes/signInAndOut/helpers.js [hunk 3]
```diff
@@ -56,37 +56,27 @@ module.exports = {
...
-                        success: true,
-                        token: doc._id,
-                    });
-                } else{
-                    return res.send({ err, message: INVALID_SIGNIN });
-                }
-            });           
+
+            await bcrypt.compare(password, user.password, (error, result) => {
+                const data = result ?
```
[the remark this answers is not in the document]
Comment (Way2nnadi) [reply routes/signInAndOut/helpers.js]: whenever you are writing code over and over again like `res.send`, you should reconsider how to re-write that code so you stay DRY(`D`on't `R`epeat `Y`ourself)

File: routes/signUp/helpers.js (hunk 2 above)
Comment (Way2nnadi) [reply routes/signUp/helpers.js]: good

File: db/models/product.js [hunk 4]
```diff
@@ -1,19 +1,15 @@
...
-// create new instance of the mongoose.schema. the schema takes an object that shows
-// the shape of your database entries.
 const ProductsSchema = new Schema({
     createdAt: { type: Date },
     deletedAt: { type: Date },
     name: { required: true, type: String },
     producerId: { required: true, type: String },
     quantity: { required: true, type: Number },
-    typeOfProduct: { required: true, type: String },
+    typeOfProducts: { required: true, type: String },
```
Comment (Way2nnadi) [inline db/models/product.js]: `typeOfProducts` should probably be an array

Comment (EngrEric) [reply db/models/product.js]: okay

**Pull request merged**